src/components/Board.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Board.css';
+import Square from './Square';
+import PropTypes from 'prop-types';
+
+
+const generateSquares = (squares, onClickCallback) => {
```

Comment:
this is the same name of the fucntion in `App`. Would you wanna rename this to something else? I'm struggling to think of a more fitting name... ugh

Reply:
Thanks! Love it